packages/cli/src/platform/$.ts

```diff
@@ -0,0 +1,20 @@
+import { Command, Commands } from '@prisma/internals'
+
+import { EarlyAccessFlagError } from '../../../migrate/src/utils/flagErrors'
+
+export class $ implements Command {
+  public static new(commands: Commands): $ {
+    return new $(commands)
+  }
+
+  private constructor(private readonly commands: Commands) {}
+
+  public async parse(argv: string[]) {
+    const isHasEarlyAccessFeatureFlag = Boolean(argv.find((_) => _.match(/early-access-feature/)))
```

Comment:
Not really new, but maybe we want to make this shorter as just `--early-access` (so without the `-feature` suffix)?

Reply:
That would be nice :)

Reply:
Let's ask @Jolg42, but as we are not using that at all (and haven't for years) I think that should be fine.

Reply:
Feel free to change it, shorter is better I think too, make sure to search everywhere for it in this repository 👍🏼

Reply:
@Jolg42 @janpio I'll make the change but in a different PR if that's ok.